modules/apps/site/site-item-selector-web/src/main/java/com/liferay/site/item/selector/web/internal/MySitesItemSelectorView.java

```diff
@@ -68,8 +70,16 @@
 
 	@Override
 	public String getTitle(Locale locale) {
+		PermissionChecker permissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+		String labelKey = "my-sites";
+
+		if (permissionChecker.isOmniadmin()) {
```

Comment:
According to PTR-2447, it sounds like it was any portal administrator (`PermissionChecker.isCompanyAdmin`) rather than just the omni admin (`PermissionChecker.isOmniadmin`) who would get a different experience. Can you confirm by creating a user with the Administrator role? If they also see all sites, can you update the check?